src/reverselock.h

```diff
@@ -13,7 +13,7 @@ class reverse_lock
 {
 public:
 
-    explicit reverse_lock(Lock& lock) : lock(lock) {
+    explicit reverse_lock(Lock& _lock) : lock(_lock) {
```

Comment:
And if you need all-in-one diff, you can use https://patch-diff.githubusercontent.com/raw/bitcoin/bitcoin/pull/8655.diff